### ISSUE TITLE
Documentation interaction with space key ?

### ISSUE THREAD

**Issue by Jacques-Olivier-Farcy** [thecodingmachine/workadventure#559]
Hi,

I saw that it is possible to interact with some object with the space key.
Is it possible to know how to implement that in Tiled ?

**Comment by moepman** [comment]
There is this computer thing I have seen in one map which is very specific for one use case.

I'd rather see this usable in a more generic way so that for example you could specify an animation and maybe wether it loops or only runs through once when you interact with the tile.

**Comment by moufmouf** [comment]
Hey!

The little computer next to my desk is a prototype of an interactive object.
So far it is fairly useless (you can turn it on or off and when turned on, there is a small animation going on).

This is in fact only a prototype. In the future, we will add support for all kind of objects.
So far, I'd rather avoid documenting this part as it is completely experimental, and subject to change.

In an ideal world, I would like users to be able to ship their own objects with their own code in the maps, but this is challenging as this comes with a number of security issues.

**Comment by moepman** [comment]
Yeah the security issues with user-supplied scripts were the reason why I suggested that for the start we could do with a few attributes such as maybe an iframe to open when interacting, or toggling an animation. That would allow for lots of use cases without needing to run custom scripts.
